### DISCUSSION TITLE
What's a recommended way to inject secrets to a Deployment?

### DISCUSSION THREAD

**Discussion by zzgab** [domcloud/domcloud#69] [Ideas]
I am running deployments using automated git clone & a GitHub Deploy Key.
My service needs some secret file (let's say a private key) which I don't want to put in GitHub.
What would be a recommended way for application to obtain this sensitive file at deploy-time? I couldn't find a "Vault" section in DOMCloud (or Secrets Manager or equivalent).
Thanks.
Gabriel

**Comment by Senophyx** [comment]
You can use `.env` on your main directory.

**Reply by willnode** [reply]
Correct. It can be inserted via SSH, VSCode or any other file managers

**Reply by zzgab** [reply]
Thanks!
